menu.py

```diff
@@ -8,55 +8,55 @@
 
 def printHeader():
     os.system('clear')
-    print 'S N A K E\n'
+    print('S N A K E\n')
 
 def tutorial():
     global state
 
     if state == 0:
         printHeader()
-        print '\nHi there! Welcome to Snake.'
-        print 'Type snake to launch the game'
+        print('\nHi there! Welcome to Snake.')
+        print('Type snake to launch the game')
         success = False
         while not success:
-            command = raw_input()
+            command = input()
             if command == 'snake':
-                print '\nUse the keys to move and Q to quit'
+                print('\nUse the keys to move and Q to quit')
                 subprocess.call("sleep 1", shell=True)
-                print '3...'
+                print('3...')
                 subprocess.call("sleep 1", shell=True)
-                print '2...'
+                print('2...')
                 subprocess.call("sleep 1", shell=True)
-                print '1...'
+                print('1...')
                 subprocess.call("sleep 1", shell=True)
                 subprocess.call("python __main__.py", shell=True)
                 success = True
                 state += 1
             else:
-                print 'Type snake to launch the game'
+                print('Type snake to launch the game')
 
     elif state == 1:
         printHeader()
-        print 'Great game. Now let\'s see what options are available.'
-        print 'Type snake --help'
+        print('Great game. Now let\'s see what options are available.')
+        print('Type snake --help')
         success = False
         while not success:
-            command = raw_input()
+            command = input()
             if command == 'snake --help' or command == 'snake -h':
                 subprocess.call("python . --help", shell=True)
                 subprocess.call("sleep 1", shell=True)
                 success = True
                 state += 1
             else:
-                print 'Type snake --help'
+                print('Type snake --help')
 
     elif state == 2:
-        print '\n\nYou can try different themes for the game'
-        print 'For example, type snake --theme minimal'
-        print 'Or type snake --help to get a list of available themes and then snake --theme "name"'
+        print('\n\nYou can try different themes for the game')
```

Comment:
Using an additional `print()` or `os.linesep`  would be more portable and (arguably) better to read than using `\n`s.

Or, let's exaggerate, we introduce a convenience function like this: :ghost: 
``` python
def say(*lines):
    for text in lines:
        print(text)

say('',
    '',
    'You can try different themes for the game',
    'For example, type snake --theme minimal',
    'Or type snake --help to get a list of available themes and then snake --theme "name"')
```